### DISCUSSION TITLE
Not able to get the output

### DISCUSSION THREAD

**Discussion by faizu6** [aquasecurity/trivy#4495] [Q&A] [answered]
### Question

I was running the trivy initially but from yesterday, it giving me hard time. I am not able to get even a single line as output. I uninstalled trivy and then installed it again but i am still facing the same issue. The screenshot I attached the command is running from last 3 minutes and not even a single line is being printed. Could you suggest the possible reason of this. Appreciate the help in advance.
![image](https://github.com/aquasecurity/trivy/assets/111133381/b8500848-b68a-4d95-9d52-db304a49206f)

### Target

Git Repository

### Scanner

Vulnerability

### Output Format

Table

### Mode

Standalone

### Operating System

ubuntu 22.04

### Version

```bash
faizan@Faizan:~$ trivy -version
Version: 0.18.3
Vulnerability DB:
  Type: Full
  Version: 2
  UpdatedAt: 2023-05-29 06:07:56.726462056 +0000 UTC
  NextUpdate: 2023-05-29 12:07:56.726461656 +0000 UTC
  DownloadedAt: 2023-05-29 06:33:14.26219616 +0000 UTC
```

**Comment by afdesk** [comment]
@faizu6 i don't see any issues this.

maybe did you run several Trivy scans in the same time?


also you use really old version. Please update your Trivy

**Comment by faizu6** [comment, marked as answer]
@afdesk thanks for the quick reply, removed the processes in the background and its working fine now.
